1626209073085	Sync.LogManager	DEBUG	Flushing file log
1626209073093	Sync.LogManager	DEBUG	Log cleanup threshold time: 1625345073093
1626209073113	Sync.LogManager	DEBUG	Done deleting files.
1626216839051	Sync.Service	DEBUG	User-Agent: Firefox/78.11.0 (Linux x86_64) FxSync/1.80.0.20210531140902.desktop
1626216839052	Sync.Service	INFO	Starting sync at 2021-07-14 08:53:59 in browser session ZUQImpKdXDFJ
1626216839053	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1626216839058	Sync.Status	INFO	Resetting Status.
1626216839058	Sync.Status	DEBUG	Status.service: error.sync.failed => success.status_ok
1626216839060	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1626216839060	FirefoxAccounts	DEBUG	getOAuthToken enter
1626216839062	FirefoxAccounts	TRACE	getCachedToken returning cached token
1626216839062	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1626216839062	Sync.BrowserIDManager	INFO	Getting keys
1626216839064	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1626216839067	Sync.BrowserIDManager	INFO	Getting an assertion from: https://token.services.mozilla.com/1.0/sync/1.5
1626216839068	FirefoxAccounts	DEBUG	enter getAssertion()
1626216839073	Services.Common.RESTRequest	DEBUG	GET https://profile.accounts.firefox.com/v1/profile failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1626216839074	FirefoxAccounts	ERROR	Background refresh of profile failed: {"name":"FxAccountsProfileClientError","code":null,"errno":998,"error":"NETWORK_ERROR","message":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 488\"  data: no]"}
1626216839075	FirefoxAccounts	DEBUG	got keyPair
1626216839075	FirefoxAccounts	DEBUG	getCertificateSigned: true true
1626216839082	Services.Common.RESTRequest	DEBUG	POST request to https://api.accounts.firefox.com/v1/certificate/sign?service=sync
1626216839082	Services.Common.RESTRequest	DEBUG	POST Length: 888
1626216839086	Services.Common.RESTRequest	DEBUG	POST https://api.accounts.firefox.com/v1/certificate/sign?service=sync failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1626216839087	Hawk	WARN	hawk request error: [Exception... "NS_ERROR_UNKNOWN_HOST"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "JS frame :: resource://services-common/rest.js :: onStopRequest :: line 488"  data: no] Stack trace: onStopRequest()@resource://services-common/rest.js:488
1626216839087	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status code: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get status :: line 624"  data: no] Stack trace: get status()@resource://services-common/rest.js:624
request()@resource://services-common/hawkclient.js:239
1626216839087	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status text: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get statusText :: line 639"  data: no] Stack trace: get statusText()@resource://services-common/rest.js:639
request()@resource://services-common/hawkclient.js:247
1626216839088	Hawk	DEBUG	(Response) /certificate/sign?service=sync: code: null - Status text: null
1626216839088	Services.Common.RESTResponse	DEBUG	Caught exception processing response headers: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get headers :: line 671"  data: no] Stack trace: get headers()@resource://services-common/rest.js:671
_maybeNotifyBackoff()@resource://services-common/hawkclient.js:316
request()@resource://services-common/hawkclient.js:255
1626216839089	Services.Common.RESTResponse	DEBUG	Caught exception processing response headers: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get headers :: line 671"  data: no] Stack trace: get headers()@resource://services-common/rest.js:671
_maybeNotifyBackoff()@resource://services-common/hawkclient.js:316
request()@resource://services-common/hawkclient.js:256
1626216839089	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status text: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get statusText :: line 639"  data: no] Stack trace: get statusText()@resource://services-common/rest.js:639
_constructError()@resource://services-common/hawkclient.js:122
request()@resource://services-common/hawkclient.js:261
1626216839089	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status code: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get status :: line 624"  data: no] Stack trace: get status()@resource://services-common/rest.js:624
_constructError()@resource://services-common/hawkclient.js:123
request()@resource://services-common/hawkclient.js:261
1626216839090	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status code: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get status :: line 624"  data: no] Stack trace: get status()@resource://services-common/rest.js:624
_constructError()@resource://services-common/hawkclient.js:124
request()@resource://services-common/hawkclient.js:261
1626216839090	Services.Common.RESTResponse	DEBUG	Caught exception processing response headers: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get headers :: line 671"  data: no] Stack trace: get headers()@resource://services-common/rest.js:671
_constructError()@resource://services-common/hawkclient.js:130
request()@resource://services-common/hawkclient.js:261
1626216839090	FirefoxAccounts	ERROR	error POSTing /certificate/sign?service=sync: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 488\"  data: no]","message":null,"code":null,"errno":null}
1626216839091	FirefoxAccounts	ERROR	HAWK.signCertificate error: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 488\"  data: no]","message":null,"code":null,"errno":null}
1626216839091	Sync.BrowserIDManager	ERROR	Non-authentication error in _fetchTokenForUser: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 488\"  data: no]","message":null,"code":null,"errno":null}
1626216839091	Sync.Status	DEBUG	Status.login: success.login => error.login.reason.network
1626216839091	Sync.Status	DEBUG	Status.service: success.status_ok => error.login.failed
1626216839092	Sync.BrowserIDManager	ERROR	Failed to fetch a token for authentication: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 488\"  data: no]","message":null,"code":null,"errno":null}
1626216839093	FirefoxAccounts	TRACE	not checking freshness of profile as it remains recent
1626216839096	Sync.BulkKeyBundle	INFO	BulkKeyBundle being created for undefined
1626216839097	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/166986188/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1626216839098	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1626216839098	Sync.Status	DEBUG	Status.service: error.login.failed => error.sync.failed
1626216839098	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1626216839098	Sync.SyncScheduler	DEBUG	Sync error count has exceeded 3; enforcing backoff.
1626216839098	Sync.SyncScheduler	DEBUG	Starting client-initiated backoff. Next sync in 12356316 ms.
1626216839098	Sync.SyncScheduler	DEBUG	Next sync in 12356316 ms. (why=client-backoff-schedule)
1626216839101	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
